CondFormats/HcalObjects/interface/HcalQIEType.h

```diff
@@ -12,6 +12,7 @@ contains the QIE Typese + corresponding DetId
 class HcalQIEType
 {
  public:
+  enum {QIE8=0, QIE10=1, QIE11=2};
```

Comment:
One of the most important 'standard' for the persistency is the fact that the CondFormat classes are frozen = no change possible. This case is actually borderline, since it does not involve persistent data. However, and especially if in the future you won't be using the enum anymore, you might consider to simply put it in a namespace... What about HcalDbHardcode ?